.github/workflows/ci.yml

```diff
@@ -3,7 +3,7 @@ on:
   pull_request:
     paths-ignore: "*.rst"
   push:
-    paths-ignore: "*.rst"
+    # paths-ignore: "*.rst"
```

Comment:
why lol *.rst changes cannot break code/tests and pre-commit errors are in a separate workflow anyway.

Reply:
There are three ___required___ GitHub Actions tests before a pull request is merged.  Without this change, a pull request that only changes .rst files can never be merged.  Failing to plan is planing to fail.

Reply:
Good point. Your change does not do anything to stop that, though, as all you changed is the part applying to pushes to the master and develop branches. I'm currently asking https://github.com/orgs/community/discussions/155654 to implement an alternative measure of path-filtering (removing the ignorance from both is also an option but I don't think we need it enough for a separate commit; I won't oppose such a change though).

* https://github.com/orgs/community/discussions/155654